app/views/content/train-to-be-a-teacher/_nav-cards.html.erb

```diff
@@ -0,0 +1,51 @@
+<section class="container">
+    <div class="category__nav-cards">
+        <a href="/train-to-be-a-teacher/if-you-have-a-degree" class="category__nav-card">
+            <h2>If you have a degree</h2>
+
+            <p>
+                Find out how to get QTS if you have a degree or you’re studying for one now. There are different options depending on your situation.
+            </p>
+        </a>
+
+        <a href="/train-to-be-a-teacher/if-you-have-experience" class="category__nav-card">
+            <h2>If you have classroom experience</h2>
+
+            <p>
+                If you have already spent some time working in the classroom, you may be able to follow the assessment only route to show you meet QTS.
```

Comment:
I don't think it is quite right to say "show you meet QTS"

I think you have to show that you meet the **standards** for QTS without the need for further training - and then you get QTS at the end of the assessment.

(Need a policy eye on this to confirm though)